src/allmydata/test/web/test_grid.py

```diff
@@ -34,6 +38,7 @@ class CompletelyUnhandledError(Exception):
     pass
 
 class ErrorBoom(object, resource.Resource):
+    @render_exception
```

Comment:
To get our desired exception handling behavior, all render methods must be decorated with this now.